packunused.cabal

```diff
@@ -64,14 +71,16 @@ source-repository head
 
 executable packunused
   main-is:             packunused.hs
+  other-modules:       Paths_packunused
   default-language:    Haskell2010
   other-extensions:    CPP, RecordWildCards
   ghc-options:         -Wall -fwarn-tabs -fno-warn-unused-do-bind
   build-depends:
     base                 >=4.5  && <4.9,
     Cabal                >=1.14 && <1.23,
-    optparse-applicative >=0.8  && <0.12,
+    optparse-applicative >=0.8,
     directory            >=1.1  && <1.3,
     filepath             >=1.3  && <1.5,
-    haskell-src-exts     >=1.13 && <1.17,
-    split                ==0.2.*
+    haskell-src-exts     >=1.13,
```

Comment:
fyi, leaving off upper bounds violates the PVP and consequently conflict with Hackage guidelines; I'll readd those after I merge this PR